tests/cpp/test_gpu_view.cpp

```diff
@@ -2344,4 +2344,37 @@ TEST_F(GpuViewTest, ExpandedBroadcast) {
   testValidate(&fusion, {actual_out_tensor}, {in_tensor}, __LINE__, __FILE__);
 }
 
+TEST_F(GpuViewTest, SplitMergePointwiseSplitMerge) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+  const std::vector<int64_t> input_shape = {12, 20};
+  DataType dtype = DataType::Float;
+  auto tv0 = makeContigTensor(input_shape.size(), dtype);
+  fusion->addInput(tv0);
+  auto tv1 = castOp(DataType::Float, tv0);
+  // root domain : (i0, i2)
+  // logi domain : (3, i0/3, 4, i2/4)
```

Comment:
nitpick: logical domain?

Reply:
yes, I used `logi` since it has the same length with `root`, will change to `logical` to avoid confusion.